examples/TestReceiveAV/TestReceiveAV.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <UseWindowsForms>true</UseWindowsForms>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.MixedReality.WebRTC" Version="1.0.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="SIPSorcery.WebSocketSharp" Version="0.0.1" />

```

Comment:
We'd rather not take dependencies on third party components unless absolutely necessary. Any chance you can use the [.NET WebSocket classes](https://docs.microsoft.com/en-us/dotnet/api/system.net.websockets.websocket) instead?

Reply:
I wish I could use the in built .NET classes but to the best of my knowledge there's no WebSocket server support outside of ASP.NET and running Kestrel or IIS Express is overkill for a sample like this. I'd be very happy to be corrected but the popularity of projects like https://github.com/sta/websocket-sharp (of which the `SIPSorcery.WebSocketSharp` is a minor fork) and others points to the same fact.

Reply:
HttpListener seems to be able to act as a WebSocket server: https://docs.microsoft.com/en-us/dotnet/api/system.net.httplistenercontext.acceptwebsocketasync. It seems to me that it should be reasonably easy to write the same logic around it, unless I am missing something. Seems to be working only on Windows 8 onward but this shouldn't be an issue for us - also it might be less performant than a bespoke server implementation but since this is a sample app we don't care too much.

Reply:
That's relying on `http.sys` which is Windows specific. The current web socket server logic used in the example is not platform specific. Is it worth changing just to avoid a dependency in an example?

Reply:
I have validated the dependency internally, sounds like it's fine! You're right, it's nicer if the sample works on all platforms.